build-system/npm-publish/write-package-files.js

```diff
@@ -105,5 +107,28 @@ async function writePackageJson(extensionVersion) {
   }
 }
 
-writePackageJson('1.0');
-writePackageJson('2.0');
+async function writeReactJs(extensionVersion) {
```

Comment:
Can we add JSDoc for all the functions in this file for documentation and to enable type-checking?

/cc @rileyajones Do you think it's worth enabling the `require-jsdoc` lint rule for `build-system` so this gets auto-populated on save?